modules/installation-vsphere-machines.adoc

```diff
@@ -129,13 +168,9 @@ $ govc vm.change -vm "<vm_name>" -e "guestinfo.afterburn.initrd.network-kargs=${
 +
 *** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*.
 *** Click *Edit Configuration*, and on the *Configuration Parameters* window, click *Add Configuration Params*. Define the following parameter names and values:
-**** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded Ignition config file for this machine type. Note for the bootstrap node, the Ignition config file must be provided in `guestinfo.ignition.config.data` in the *Configuration Parameters* window. This is due to a restriction in the maximum size of data that can be provided in a vApp property.
+**** `guestinfo.ignition.config.data`: Locate the base-64 encoded files that you created previously in this procedure, and paste the contents of the base64-encoded Ignition config file for this machine type. Note that for the bootstrap node, the Ignition config file must be provided in `guestinfo.ignition.config.data` in the *Configuration Parameters* window. This is due to a restriction in the maximum size of data that can be provided in a vApp property.
```

Comment:
Since bootstrap ignition config file has already been changed to be provided by `merge-bootstrap.64`, do we still need to describe it here? Or merge-bootstrap.64 is just an option for user who could not copy/paste bootstrap based64-encoded ignition config directly?

Reply:
Good point. Seems we don't need to describe the bootstrap Ignition config note here, so I've removed it.